comfy_cli/command/run_cli.py

```
@@ -0,0 +1,579 @@
+        return (idx, None, time.time() - t0)
+    elapsed = time.time() - t0
+    if result.returncode != 0:
+        return (idx, None, elapsed)
+    try:
+        doc = json.loads(result.stdout.strip().splitlines()[-1])
+        return (idx, doc.get("data", {}).get("prompt_id"), elapsed)
+    except (json.JSONDecodeError, IndexError):
+        return (idx, None, elapsed)
+
+
+def _fleet_step(state: _DemoState, pause_seconds: float, show_agent: bool) -> int:
```

Pylint (4.0.5)

[refactor] Too many local variables (28/15)

(R0914)